client/src/util/fetchFromAPI.js

```diff
@@ -0,0 +1,25 @@
+export const queryAPI = async (options) => {
+  const { url, onSuccess, onError, method } = options;
+  options = {
+    url: "http://localhost:5000/menu_items/",
+    body: null,
+    onSuccess: (result) => console.log(result.data),
+    onError: (result) => console.log(result.error),
+    method: "GET",
+  };
```

Comment:
We don't want to re-assign a value to `options` here, meaning that this should be deleted.

```suggestion
```

Reply:
I _would_ expect to see this options definition in the calling code, if that helps?  This is probably a great time to shift focus to your test file so you can see what this function looks like on the calling side.

Reply:
10-4